src/main/java/io/cdap/plugin/gcp/bigquery/sink/BigQuerySink.java

```diff
@@ -92,6 +107,18 @@ protected BigQuerySinkConfig getConfig() {
   protected void prepareRunValidation(BatchSinkContext context) {
     FailureCollector collector = context.getFailureCollector();
     config.validate(context.getInputSchema(), config.getSchema(collector), collector);
+    try {
+      config.validatePartitionProperties(context.getInputSchema(), config.getSchema(collector), collector);
```

Comment:
this wasn't being called before, are we adding validation in this PR? The title indicates it's just about not failing deployment for certain types of validations.

Reply:
Same as above. I separated out the validatePartitionProperties helper method which used to be private and called in config.validate. This way we are still running validation logic, but not failing immediately if connection problems arise.